scripts/extract-paired-reads.py

```diff
@@ -51,6 +65,19 @@ def get_parser():
     parser.add_argument('infile')
     parser.add_argument('--version', action='version', version='%(prog)s ' +
                         khmer.__version__)
+
+    parser.add_argument('-o', '--output-dir', metavar="output_directory",
+                        dest='output_directory', default='', help='Output '
```

Comment:
Ideally the long form of an option would match the desired `args.name` which obviates the need to manually specify the `dest`. The `metavar` is used to override the `dest` name for aesthetic reasons. To my eyes the two lines above are rather redundant as ArgParse will convert dashes to underscores when generating the property name for this argument.

`parser.add_argument('-o', '--output-directory', default='',` and on as before should do the trick.